addon/utils/load-relationship.js

```diff
@@ -11,14 +11,15 @@ export default function loadAll(model, relationship, dest, options = {}) {
 
     return model.query(relationship, query).then(results => {
         dest.pushObjects(results.toArray());
-
-        var total = results.meta.pagination.total;
-        var pageSize = results.meta.pagination.per_page;
-        var remaining = total - (page * pageSize);
-        if (remaining > 0) {
-            query.page = page + 1;
-            query['page[size]'] = pageSize;
-            return loadAll(model, relationship, dest, query);
+        if (results.meta) {
+            var total = results.meta.pagination.total;
+            var pageSize = results.meta.pagination.per_page;
```

Comment:
It seems reasonable that a user depaginating the API might want to start by fetching more than the default minimum per page, by passing in the `query` initially.

As written, it seems like the "get next page after first" call would overwrite the original query page size. Eg the user could fetch 50 results the first page (because that's what they said to get), but only 10 thereafter (or whatever the API says to get next). 

Is overriding the user-provided `query` the intended behavior? Or do we want this to be a default value if none was specified explicitly?